.gitignore

```diff
@@ -0,0 +1,2 @@
+.mvn
```

Comment:
`.mvn` folder must be exist. Can you remove this line? Also, include `maven-wrapper.properties`